docs/src/cli/cmd/clone.rst

```diff
@@ -14,7 +14,7 @@ Syntax
 
 ::
 
-    titan clone <uri> [repository]
```

Comment:
Can you explain here (and perhaps in the local overview section as well) the new URI format and opetion to have "#" in the clone command? I believe there are a few places that mention "Clone always uses the latest commit ...."